### PR TITLE
Tags

### PR DESCRIPTION
**Pull request by dshanske**
This adds support for basic person tagging and improves the presentation for homepage mentions by changing the comment excerpt if the post type is not post.

### CODE REVIEW

**Comment by dshanske** [comment]
Ooops, broke a unit test.

**Comment by dshanske** [comment]
Didn't notice that the unit test didn't attach its test likes to a post. Fixed that now.

File: includes/class-linkbacks-handler.php [hunk 1]
```diff
@@ -201,29 +201,29 @@ public static function update_meta( $comment_id, $commentdata ) {
 	public static function get_comment_type_excerpts() {
 		$strings = array(
 			// translators: Name verb on domain
-			'mention'       => __( '%1$s mentioned %2$s on <a href="%3$s">%4$s</a>.', 'semantic-linkbacks' ),
+			'mention'         => __( '%1$s mentioned %2$s on <a href="%3$s">%4$s</a>.', 'semantic-linkbacks' ),
```
Comment (pfefferle) [inline includes/class-linkbacks-handler.php]: please avoid these changes in the future... it is very noisy

**Comment by pfefferle** [comment]
This only works in english, because you always use `the` or `this`, in german you have `der`, `die`, `das`, depending on the subject. That is why I implemented this function in the first place instead of using it like you did as fallback: https://github.com/pfefferle/wordpress-semantic-linkbacks/blob/master/includes/class-linkbacks-handler.php#L263

I would prefer to add filters to all of the string functions and extend the correct strings in the respective plugins, for example post-kinds or ...

**Comment by pfefferle** [comment]
I would love to merge the person tagging but not the `get_post_type` refactoring.

File: includes/class-linkbacks-handler.php (hunk 1 above)
Comment (dshanske) [reply includes/class-linkbacks-handler.php]: I think it was the phpcbf run.

**Comment by dshanske** [comment]
I have to ask about how I can address that. Can we do at least 'this page' and sitename as outlined and avoid custom post types for now?

**Comment by dshanske** [comment]
The reason I put them together is that they are both issues with tagging. One display the other Microformats.

**Pull request closed without merging**